apps/frontend-manage/src/pages/index.tsx

```diff
@@ -64,6 +71,26 @@ function Index() {
   const [isQuestionCreationModalOpen, setIsQuestionCreationModalOpen] =
     useState(false)
 
+  const dropdownItems = [
+    { value: 'CREATED', label: 'Datum' },
+    { value: 'TITLE', label: 'Titel' },
+  ]
+
+  const [sortBy, setSortBy] = useState('')
+
+  const sortIcon = useMemo(() => {
+    if (!sortBy) {
+      return faSort
+    }
+
+    if (sort.asc) {
+      return faSortAsc
+    }
+
+    return faSortDesc
+  }, [sortBy, sort.asc])
```

Comment:
To emphasize that we are sorting a list here, we could maybe use more explicit icons like faArrowUpWideShort or similar?

Reply:
see above